metabase/metabase
# Fix new questions don't appear in the Saved Questions picker

**EDIT:** the solution changed from a simple reload to adding custom reducers to schema and table entities

Fixes #9027: new questions don't appear in the Saved Questions picker until you refresh the browser. The fix is very straightforward: it makes the picker refresh the list of selected collection questions any time we select a collection

### To Verify

1. (in a new window) Custom question > Saved Questions - wait for the list of "Our analytics" to load
2. Native query > Sample Dataset > `select 0` run and save as "0-test" in "Our analytics"
3. Custom question > Saved Questions > you should see "0-test" inside "Our analytics" immediately
